lib/job_tomate/actions/slack_notify_jira_bug_issue_without_cause_updated.rb

```diff
@@ -0,0 +1,30 @@
+require "data/user"
+require "support/service_pattern"
+
+module JobTomate
+  module Actions
+
+    # Notifies the JIRA issue's assignee on Slack.
+    class SlackNotifyJIRABugIssueWithoutCauseUpdated
```

Comment:
i meant that every bug should should have a cause updated, but you're right that technically it's the issue that is updated and there's no cause. Anyway, done :)